GooglePlayInstant/LoadingScreen/LoadingBarGenerator.cs

```diff
@@ -22,20 +22,23 @@ namespace GooglePlayInstant.LoadingScreen
     /// </summary>
     public static class LoadingBarGenerator
     {
-        // Loading bar size as a proportion of the screen size. Adjust if needed.
-        private const float LoadingBarWidthProportion = 0.7f;
-        private const float LoadingBarHeightProportion = 0.02f;
+        // Loading bar size.
+        private const float LoadingBarWidth = 672;

```

Comment:
Nit: 672.0f (also 54.0f)

Reply:
Would 672f and 54f also work?

Reply:
As a personal style preference I always include the ".0", but it's a personal preference, so I'm OK without it.

Reply:
Cool, I'll include the ".0"